### PR TITLE
LPS-130796 Update `@liferay/npm-scripts` to v44.0.0

### PR DESCRIPTION
**Pull request by wincent**
[Release notes](https://github.com/liferay/liferay-frontend-projects/releases/tag/npm-scripts%2Fv44.0.0).

Most noteworthy:

- We are enforcing the use of the `@liferay` named scope to guard against dependency confusion attacks. We haven't actually renamed all the packages yet, but we have reserved placeholder packages with those names to keep them safe. In the meantime, we have an "allow list" internally that stops our formatter from complaining about those names only.

⚠️ **But note: Will need to bump 4 theme versions due to a little mishap when publishing placeholder packages, [as explained here](https://github.com/liferay/liferay-frontend-projects/pull/500#issuecomment-822615093).**

The affected themes are:

- `apps/commerce/commerce-theme-minium/commerce-theme-minium`
- `apps/commerce/commerce-theme-speedwell/commerce-theme-speedwell`
- `apps/frontend-theme/frontend-theme-styled`
- `apps/frontend-theme/frontend-theme-unstyled`

Basically, we need to bump the "patch" version component of each of those, or the next attempt at publishing them will fail. I've already messaged Brian via Slack to let him know, in case this interferes with the release.

### CODE REVIEW

**Comment by wincent** [comment]
ci:reevaluate:1217401_7908

**Comment by wincent** [comment]
Failures look unrelated, so going to retry.

**Comment by wincent** [comment]
ci:forward

**Comment by wincent** [comment]
Different unique failures, I think it's manual forward time. Any objections?

**Comment by wincent** [comment]
ci:reevaluate:1217401_7911

**Comment by izaera** [comment]
Errors look unrelated: one is due to a session expiration, the other an exception in an OSGi bundle activate method.

**Comment by wincent** [comment]
> Errors look unrelated: one is due to a session expiration, the other an exception in an OSGi bundle activate method.

Now we're in this together...

in-this-together

Manual forward: https://github.com/brianchandotcom/liferay-portal/pull/100917
